review_practice/api-server/database/models/index.js

```diff
@@ -0,0 +1,37 @@
+import fs from 'fs';
+import path from 'path';
+import Sequelize from 'sequelize';
+import configs from '../../config/database.js';
+
+const basename = path.basename(__filename);
+const env = process.env.NODE_ENV || 'development';
+const config = configs[env];
+
+const db = {};
+
+let sequelize;
+if (config.use_env_variable) {
+    sequelize = new Sequelize(process.env[config.use_env_variable], config);
+} else {
+    sequelize = new Sequelize(config.database, config.username, config.password, config);
+}
+
+fs.readdirSync(__dirname)
+    .filter(file => {
+        return file.indexOf('.') !== 0 && file !== basename && file.slice(-3) === '.js';
```

Comment:
```suggestion
        return !file.startWith('.') && file !== basename && file.endWith('.js');
```
이렇게 `startWith`와 `endWith`라는 메소드를 사용해도 된답니다!